azure-pipelines.yml

```diff
@@ -24,7 +24,7 @@ variables:
   value: "0.1" # update between releases
 - name: SEMANTIC_VERSION
   value: "$(MAJOR_MINOR_VERSION).$(Build.BuildId)"
-- name: ACR_NAME
+- name: $(APP_NAME)ACR_NAME
```

Comment:
this is invalid - did you mean to use it $() in the value field?

Reply:
Uups, I made a mistake copying sorry